hbase_storage/src/main/java/com/splicemachine/access/hbase/H10PartitionAdmin.java

```diff
@@ -508,4 +508,22 @@ private UserPermission getPermission(List<UserPermission> permissions, String us
         }
         return null;
     }
+
+    private void retriableMergeRegions(String regionName1, String regionName2, int maxRetries, int retry) throws IOException {
+        try {
+            admin.mergeRegions(Bytes.toBytes(regionName1), Bytes.toBytes(regionName2), false);
+        }
+        catch (MergeRegionException e) {
+            SpliceLogUtils.warn(LOG, "Merge failed:", e);
+            if (e.getMessage().contains("Unable to merge regions not online") && retry < maxRetries) {
```

Comment:
It seems that if merge fails after maxRetries, we won't throw any exception to the caller, is this an intended behavior? Thanks.